build.gradle

```diff
@@ -9,7 +9,7 @@
 buildscript {
     ext.android_support_version = '28.0.0'
     ext.kotlin_version = '1.3.0'
-    ext.android_components_version = '0.40.0'
+    ext.android_components_version = '0.51.0-SNAPSHOT'
```

Comment:
>may have something to say about merge dates, since this PR upgrades @mozilla-mobile/android-components to a snapshot version.

Thanks @jhugman. Yes, good point. We quickly discussed waiting and I believe 0.51.0 should go out as a final [release](https://github.com/mozilla-mobile/android-components/releases) today? So we can wait for that, bump this, then re-test and merge 🤞

Reply:
Unfortunately, I found out yesterday that, IIUC, no a-c release is going out this week, https://mozilla.slack.com/archives/CA4JKKLSG/p1555962016115900

You can probably take it up with that team, and they might change their mind, I didn't press the issue too much when they told me.

Reply:
>Unfortunately, I found out yesterday that, IIUC, no a-c release is going out this week

AHA! Thanks for the context @thomcc.

@eliserichards @sashei is this OK by you to keep un-merged until next week? Or would you like to merge now and build/test against it for a few days and then file a follow-up to jump off the "SNAPSHOT" for later next week?

Reply:
I'm fine keeping it un-merged until the next a-c release.